### DISCUSSION TITLE
Next DocumentServer Release

### DISCUSSION THREAD

**Discussion by rdenarie** [ONLYOFFICE/DocumentServer#2740] [General]
Hello,

We integrate OnlyOffice Document Server currently, and we plan to update to the last available version in the coming weeks.

As I can see, the last release was done in frebruary (8.0.1), and in the past releases was done each 1-3 months. 

Have you a plan for your next release ? I wish to know if I should wait few weeks/months to make my upgrade, or if you plan it in a long time, and so, start my upgrade on 8.0.1.

Thanks a lot for your answer.

**Comment by ibnpetr** [comment]
Hello, 

> in the past releases was done each 1-3 months

Absolutely!
The release is now planned for the latter part of June, but let's just say it's towards the end of June to be safe. Hopefully, this info helps you make the best choice!

**Reply by rdenarie** [reply]
Perfect ! 
Thanks for your quick answer :)
